### DISCUSSION TITLE
[General] <Is there a tutorial for setting up a zkSync Era node server?>

### DISCUSSION THREAD

**Discussion by ManutdGTA** [zkSync-Community-Hub/zksync-developers#545] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Is there a tutorial for setting up a zkSync Era node server?
Is there a node snapshot available?

this is not easy to learn
https://docs.zksync.io/infra/running-node.html#redeploying-the-zksync-node-with-a-new-pg-dump

**Comment by bxpana** [comment, marked as answer]
You'll want to start here for running a node: https://docs.zksync.io/infra/quickstart.html

Right now the Node is read only so you would use this node to read information from the zkSync Era network.

**Reply by ManutdGTA** [reply]
thx for the reply,the Node is read only,so I cant send transaction with node I run
